README.md

```diff
@@ -20,10 +23,38 @@ npm install @node-saml/passport-saml
 
 ## Usage
 
-The examples utilize the [Feide OpenIdp identity provider](https://openidp.feide.no/). You need an account there to log in with this. You also need to [register your site](https://openidp.feide.no/simplesaml/module.php/metaedit/index.php) as a service provider.
+The examples utilize the [Feide OpenIdp identity provider](https://openidp.feide.no/). You need an
+account there to log in with this. You also need to
+[register your site](https://openidp.feide.no/simplesaml/module.php/metaedit/index.php) as a service
+provider.
 
 ### Configure strategy
 
+**Most of the configuration options for the Strategy constructor are simply passed through to the
+underlying `node-saml` library. For more details on the configuration options and how the underlying
+SAML flows work, please see the
+[node-saml documentation](https://github.com/node-saml/node-saml/blob/master/README.md)**
+
+#### Config parameter details
+
+These are only the additional configuration parameters related to `passport-saml`. For the full list
+of parameters, please see the
+[node-saml documentation](https://github.com/node-saml/node-saml/blob/master/README.md)
```

Comment:
This really an internet-wide problem with software, where you have one version installed, but then look up the docs for the software on the web. Either you get the latest docs and not your current docs, or it's up to you to track down the docs for the version of the software you actually need. 

In summary: I think this is "good enough" without further changes and is on-par with what other software projects do.

Reply:
@srd90 , you're talking about "Release Branch Workflow" vs. "Continuous Delivery". We currently use a "Continuous Delivery" methodology, and I've used "Release Branch Workflow" before. I only think that partially solves the problem because you then have to make sure you keep the README up-to-date.

My goal is to make sure that we release `passport-saml` shortly after a release to `node-saml` to keep things in semver-major lock-step. When it comes to patch levels that may add a new config value, we might actually want to point to the new README because that would contain the new feature, which is 100% compatible with `passport-saml`. Pointing to an older README in that case would actually do the consumer a disservice.